zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/util/AuthorizationsUtil.java

```diff
@@ -116,7 +117,11 @@ public void awaitUserExistsInElasticsearch(final String username) {
                     }
                   }"""
                           .formatted(username)))
-              .uri(new URI("http://%s/identity-users-8.7.0_/_count/".formatted(elasticsearchUrl)))
+              .uri(
+                  new URI(
```

Comment:
@ChrisKujawa this change has an impact on the failing IT test. It might be the reason why the test is failing

Reply:
Ideally we will change this and just send a request to the API. This is legacy from before that was available 😄 Not saying you need to do this in this PR, changing the hardcoded index would work for now I suppose.

Reply:
@remcowesterhoud @nicpuppa I dont think this is related.

1. It will definitely use the right index, with this approach
2. When debugging it is not going threw this
3. When changing it back it also doesn't work